src/lib/util/tauri.ts

```diff
@@ -54,3 +54,13 @@ export async function resetAndClear({
     errorLogger(error);
   }
 }
+
+export async function writeFile(
+  configFullPath: string,
+  config: IConfig
+): Promise<void> {
+  await invoke('create_file', {
+    path: configFullPath,
+    content: JSON.stringify(config, null, 2)
+  }).catch((error) => errorLogger(error));
```

Comment:
I am changing error handling in another PR, but moving forward instead of catch and log it is better to throw an error and update store, so it is displayed on UI

Reply:
We can update here accordingly if your PR merges first